mobile/lib/utils/supabase_options.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:supabase_flutter/supabase_flutter.dart';
+
+Future<void> initializeSupabase() async {
```

Comment:
This would cause a merge conflict. Please normally pull from the flutter_ke branch before making a pull request

Also there's instructions on how to use the .env file to pass in the supabase credentials. They are in the README on supabase